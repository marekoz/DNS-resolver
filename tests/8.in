-s 1.1.1.1 -6  google.com 