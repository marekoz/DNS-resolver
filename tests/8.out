Authoritative: No, Recursive: No, Truncated: No
Question section (1)
  google.com., Type: AAAA, Class: IN
Answer section (1)
  google.com., Type: AAAA, Class: IN, TTL: <number>, Data length: 16, 2a00:1450:4014:080e:0000:0000:0000:200e
Authority section (0)
Additional section (0)